src/test/resources/pages/instructorFeedbackAllSessionTypes.html

```diff
@@ -758,6 +758,8 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
       <input name="user" type="hidden" value="CFeedbackUiT.instructor">
       <input name="token" type="hidden" value="${sessionToken}">
     </form>
+    <div data-value="" name="editsessiondetails">
```

Comment:
@whipermr5 this is empty because we pass an optional attribute down to the `feedbackSessionsForm` tag while loading the instructorFeedbackEditPage. Wherever the value is empty, the page being loaded is the `instructorFeedbackSessionsPage`, which does not get a value for the optional attribute. Hence it is empty. (See https://github.com/TEAMMATES/teammates/pull/7615/commits/55e1f3f73c9a237dea8790c758f96248342ac222 for the optional attribute being added)